.github/tools

```diff
@@ -1,6 +1,9 @@
 controller-gen v0.7.0
 kustomize v3.9.4
-oc v4.9.7
+oc v4.8.11
```

Comment:
Had to do this since mirror was trying to install this version and tools cache validation was failing without this

```bash
+ rm -rf /tmp/tmp.sFQfPvVtFW
+ [[ -f /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc ]]
+ OS=linux
+ ARCH=amd64
+ curl -sSLo /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc https://mirror.openshift.com/pub/openshift-v4/amd64/clients/oc/latest/linux/oc.tar.gz
+ tar -xf /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc -C /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin
+ rm -f /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/README.md
+ /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc version
Client Version: 4.8.11
+ version=v4.9.7
+ /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc version
+ grep -q 4.9.7
make: *** [Makefile:132: /home/runner/work/monitoring-stack-operator/monitoring-stack-operator/tmp/bin/oc] Error 1
```

Reply:
The right way to do this is to edit the makefile